assets/models/PhosphateChunkApatiteLarge1.tscn

```diff
@@ -1,31 +1,97 @@
-[gd_scene load_steps=10 format=3 uid="uid://ork3xiril0bq"]
+[gd_scene load_steps=12 format=3 uid="uid://ork3xiril0bq"]
 
-[ext_resource type="Material" uid="uid://b056l0x46ynl0" path="res://assets/models/CrystalApatite.material" id="1_gaqkn"]
+[ext_resource type="Shader" path="res://shaders/Crystal.gdshader" id="1_0qsxg"]
 [ext_resource type="Texture2D" uid="uid://baxuoyeo83r2u" path="res://assets/textures/dissolve_noise.tres" id="2_w0d41"]
 [ext_resource type="Script" path="res://src/microbe_stage/OrganelleMeshWithChildren.cs" id="3_b8lrv"]
 [ext_resource type="Texture2D" uid="uid://c4givkmxrlllx" path="res://assets/textures/Crystal.png" id="3_hcl7f"]
-[ext_resource type="Shader" path="res://shaders/IronChunk.gdshader" id="4_w85ja"]
+[ext_resource type="Texture2D" uid="uid://dvp0yex8rwygn" path="res://assets/textures/Crystal_rough_emit.png" id="4_7gni5"]
 [ext_resource type="ArrayMesh" uid="uid://33tvx663lr0x" path="res://assets/models/PhosphateChunkApatiteLargeCrystal.mesh" id="5_cdma8"]
 [ext_resource type="Texture2D" uid="uid://ppco0jrr2v0q" path="res://assets/textures/Crystal_normal.png" id="7_1vk26"]
 [ext_resource type="ArrayMesh" uid="uid://bxhmnyjk1vfrh" path="res://assets/models/PhosphateChunkApatiteLargeRock.mesh" id="8_7qf82"]
 
+[sub_resource type="ShaderMaterial" id="ShaderMaterial_ha8r3"]
+resource_local_to_scene = true
+render_priority = 18
+shader = ExtResource("1_0qsxg")
+shader_parameter/albedo = Color(0.12549, 0.282353, 0.905882, 0.956863)
+shader_parameter/useNormal = false
+shader_parameter/roughness = 0.6
+shader_parameter/roughnessTextureChannel = Vector4(1, 0, 0, 0)
+shader_parameter/specular = 0.7
+shader_parameter/metallic = 0.57
+shader_parameter/metallicTextureChannel = Vector4(1, 0, 0, 0)
+shader_parameter/emission = Color(0.00715748, 0.134014, 0.652016, 0.956863)
+shader_parameter/emissionEnergy = 0.75
+shader_parameter/clearcoat = 0.3
+shader_parameter/clearcoatRoughness = 0.1
+shader_parameter/dissolveValue = 0.0
+shader_parameter/outlineWidth = 0.01
+shader_parameter/growColor = Color(0.12549, 0.388235, 0.643137, 1)
+shader_parameter/albedoTexture = ExtResource("3_hcl7f")
+shader_parameter/roughnessTexture = ExtResource("4_7gni5")
+shader_parameter/dissolveTexture = ExtResource("2_w0d41")
+
+[sub_resource type="Shader" id="Shader_ixbn6"]
```

Comment:
Shaders should be saved as separate files.

Reply:
That was actually just the iron shader. I'm not sure how it ended up like that.

Reply:
Did you push your latest changes because I'm not seeing the change to emitting some phosphate? And this embedded shader code is still in this file.

Reply:
Updated now.